### PR TITLE
Alpheios release 3.0

### PR DESCRIPTION
**Pull request by balmas**
with the release up, I'd like to merge and close the branch.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,11 +32,11 @@
...
-    "@babel/core": "^7.5.5",
-    "@babel/plugin-transform-runtime": "^7.5.5",
-    "@babel/runtime": "^7.5.5",
-    "alpheios-components": "github:alpheios-project/components",
-    "alpheios-node-build": "github:alpheios-project/node-build",
+    "@babel/core": "^7.6.0",
+    "@babel/plugin-transform-runtime": "^7.6.0",
+    "@babel/runtime": "^7.6.0",
+    "alpheios-components": "github:alpheios-project/components#v1.2.45",
+    "alpheios-node-build": "github:alpheios-project/node-build#v0.6.3",
```
Comment (kirlat) [inline package.json]: Probably we don't need dependencies fixed here too

File: src/background/background-process.js [hunk 2]
```diff
@@ -75,7 +75,7 @@ export default class BackgroundProcess {
   }
 
   updateIcon (active, tabId) {
-    let params = { path: active ? this.browserIcons.active : this.browserIcons.nonactive }
+    const params = { path: active ? this.browserIcons.active : this.browserIcons.nonactive }
```
Comment (irina060981) [inline src/background/background-process.js]: I believe that we should use `let` here - as we could change properties in the following line.

File: src/background/background-process.js [hunk 3]
```diff
@@ -218,11 +219,11 @@ export default class BackgroundProcess {
 
   /**
    * Creates a TabScript object and loads content script(s) into a corresponding browser tab
-   * @param {Number} tabID - An ID of a tab
+   * @param {object} tabObj - An object that represents a tab
    * @return {Promise.<TabScript>} A Promise that is resolved into a newly created TabScript object
    */
   async createTab (tabObj) {
-    let newTab = new TabScript(tabObj)
+    const newTab = new TabScript(tabObj)
```
Comment (irina060981) [inline src/background/background-process.js]: The same about `const`

File: src/background/background-process.js [hunk 4]
```diff
@@ -268,10 +269,10 @@ export default class BackgroundProcess {
    * @return {Promise} Will be resolved when all scripts are loaded successfully.
    */
   loadContentData (tabScript) {
-    let polyfillScript = this.loadPolyfill(tabScript.tabObj.tabId)
-    let contentScript = this.loadContentScript(tabScript.tabObj.tabId)
-    let contentCSS = []
-    for (let fileName of this.settings.contentCSSFileNames) { // eslint-disable-line no-unused-vars
+    const polyfillScript = this.loadPolyfill(tabScript.tabObj.tabId)
+    const contentScript = this.loadContentScript(tabScript.tabObj.tabId)
+    const contentCSS = []
```
Comment (irina060981) [inline src/background/background-process.js]: We changing this variable - than `const` is not appropriate here.

File: src/background/background-process.js (hunk 2 above)
Comment (balmas) [reply src/background/background-process.js]: ok, will override the linter

File: src/background/background-process.js (hunk 3 above)
Comment (balmas) [reply src/background/background-process.js]: ok, will override the linter

File: src/background/background-process.js (hunk 4 above)
Comment (balmas) [reply src/background/background-process.js]: ok, will override the linter

**Pull request merged**